pkgs/by-name/md/mdbook-variables/package.nix

```diff
@@ -0,0 +1,26 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitLab,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "mdbook-variables";
+  version = "0.2.4";
+
+  src = fetchFromGitLab {
+    owner = "tglman";
+    repo = "mdbook-variables";
+    rev = version;
+    hash = "sha256-whvRCV1g2avKegfQpMgYi+E6ETxT2tQqVS2SWRpAqF8=";
+  };
+
+  cargoHash = "sha256-uw1oWIoKi6qsObI4SkEiHwEj9QoxE9jufu9O+ZKM8DI=";
+
```

Comment:
Would be better to add https://nixos.org/manual/nixpkgs/unstable/#versioncheckhook